build/cni/Makefile

```diff
@@ -22,13 +22,14 @@ CNI_RELEASE?=07a8a28637e97b22eb8dfe710eeae1344f69d16e
 CNI_TARBALL=cni-$(ARCH)-$(CNI_RELEASE).tar.gz
 CUR_DIR=$(shell pwd)
 OUTPUT_DIR=$(CUR_DIR)/output
+GOLANG_VERSION=1.7
```

Comment:
I think we might still want to fix this. can we make this `golang:1.7.4` explicitly?